station/planner.py

```diff
@@ -63,7 +122,11 @@ def execute(interval, cron=None):
 
     passes = get_passes(prediction_config, start, end)
     clear(cron)
```

Comment:
The dry run clears the crontab. It shouldn't provide any changes.

Reply:
Ok, this is now fixed.